cmd/podman/generate/systemd.go

```diff
@@ -97,10 +103,31 @@ func init() {
 	flags.StringVar(&format, formatFlagName, "", "Print the created units in specified format (json)")
 	_ = systemdCmd.RegisterFlagCompletionFunc(formatFlagName, common.AutocompleteFormat(nil))
 
+	flags.StringArrayVar(&wants, wantsFlagName, []string{}, "Systemd wants(network-online.target is default and couldn't be removed or rewritten)")
```

Comment:
```suggestion
	flags.StringArrayVar(&wants, wantsFlagName, []string{}, "Systemd wants (network-online.target is default and couldn't be removed or rewritten)")
```

> network-online.target is default and couldn't be removed or rewritten

I don't understand what that means (same below).

Reply:
Yeah, recommend rephrasing to "Add dependencies to the generated unit file" or similar

Reply:
Thank you. Fixed with the new description